packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -82,14 +82,18 @@ function st(fragment: Element, key: Key): VStatic {
 
 // [fr]agment node
 function fr(key: Key, children: VNodes, stable: 0 | 1): VFragment {
+    const leading = t('');
+    const trailing = t('');
     return {
         type: VNodeType.Fragment,
         sel: undefined,
         key,
         elm: undefined,
-        children: [t(''), ...children, t('')],
+        children: [leading, ...children, trailing],
         stable,
         owner: getVMBeingRendered()!,
+        leading,
+        trailing,
```

Comment:
This is great! I love being explicit here, instead of `elm` implicitly being either leading or trailing. ❤️